akka-docs/src/main/paradox/testing.md

```diff
@@ -388,16 +388,16 @@ Java
 :   @@snip [TestKitDocTest.java](/akka-docs/src/test/java/jdocs/testkit/TestKitDocTest.java) { #test-special-probe }
 
 You have complete flexibility here in mixing and matching the `TestKit`
-facilities with your own checks and choosing an intuitive name for it. In real
+facilities with your checks and choosing an intuitive name for it. In real
 life your code will probably be a bit more complicated than the example given
 above; just use the power!
 
 @@@ warning
 
 Any message sent from a `TestProbe` to another actor which runs on the
-CallingThreadDispatcher runs the risk of dead-lock, if that other actor might
+CallingThreadDispatcher runs the risk of dead-lock if that other actor might
 also send to this probe. The implementation of `TestProbe.watch` and
-`TestProbe.unwatch` will also send a message to the watchee, which
+`TestProbe.unwatch` will also send a message to the watcher, which
```

Comment:
I think it's actually the watchee that is meant here? This is 'the actor that is being watched', perhaps that's obscure language
```suggestion
`TestProbe.unwatch` will also send a message to the actor being watched, which
```